### DISCUSSION TITLE
'*vatr' or '*vatr*' ?

### DISCUSSION THREAD

**Discussion by MM25Zamanian** [AliMD/alwatr#84] [Q&A] [answered]
[What is it? ](https://github.com/AliMD/vatr/blob/next/package/logger/src/logger.ts#L59)

**Comment by alimd** [comment]
واقعا اینا چرا به عنوان ایشو باز شده؟!
ببین تو همون پول رکوئست که بسته شده شما میتونی زیر هر خط کامنت بزاری و منو هم منشن کنی
اگر هم جایی لینک میدی لطفا خط رو انتخاب کنی و کپی پرمالینک بزن
اینجوری من نمی‌فهمم کدوم خط رو میگی

**Reply by MM25Zamanian** [reply]
توی لینک رو ببینید آخرش زده خب چندم
و اینکه اگر با سیستم باز کنید میبینید که خط رو مشخص میکنه

**Reply by alimd** [reply]
من رو موبایل پس نمی‌بینم!

**Reply by MM25Zamanian** [reply]
با سیستم دقیق نشون میده کدوم خط رو سلکت کردم و روش ایشو باز کردم

**Comment by alimd** [comment, marked as answer]
ما * آخر رو در نظر نمیگیریم و منظورمون همونه یعنی خلاصه هر جای متن بود درسته پس عملا میشه *vatr*

**Reply by MM25Zamanian** [reply]
واسع مثال بهتره تهش نداشته باشه که کاملا واضح باشه
